src/routes/TraderForm/Form/SwapConfirmationFlow/SwapDetails/index.tsx

```diff
@@ -25,6 +26,14 @@ export const SwapDetails: React.FunctionComponent<SwapDetailsProps> = () => {
   const fixedRate = formatNumber(infoPostSwap.value.averageFixedRate);
   const variableRate = formatNumber(variableRateInfo.value);
 
+  const notional =
+    prospectiveSwapNotional < 1
```

Comment:
to be honest this logic < 1 is quite good but execution is poor. It is repeated across places.

Why not move all this logic to selectors and over there we can also add small helper function, maybe call it swapFormCompactFormat, and we can use it across all selectors. Then our UI code is less complex and just selects from the state.

Reply:
[05fe837](https://github.com/Voltz-Protocol/voltz-ui/pull/587/commits/05fe837b4cb3abd9425e03bd5049214e3c9c7ea7)